cmd/root.go

```diff
@@ -112,13 +109,40 @@ func beginDetectNewVersion() chan interface{} {
 		if local.OlderThan(*remote) {
 			executablePathSegments := strings.Split(strings.Replace(os.Args[0], "\\", "/", -1), "/")
 			executableName := executablePathSegments[len(executablePathSegments)-1]
-			log.Info("beginDetectNewVersion: A newer version of Blobfuse2 is available. Current Version=%s, Latest Version=%s", common.Blobfuse2Version, remoteVersion)
-			fmt.Fprintf(stderr, "\n\t*** "+executableName+": A newer version %s is available. Kindly update to latest version for bug-fixes & new features. ***\n\n", remoteVersion)
+			log.Info("beginDetectNewVersion: A new version of Blobfuse2 is available. Current Version=%s, Latest Version=%s", common.Blobfuse2Version, remoteVersion)
+			fmt.Fprintf(stderr, "*** "+executableName+": A new version (%s) is available. Consider upgrading to latest version for bug-fixes & new features. ***\n", remoteVersion)
+
+			_, isPresent := vJson.SecurityWarnings[common.Blobfuse2Version]
+			if isPresent {
+				hasWarning := false
+				ctr := 1
+				for _, msg := range vJson.SecurityWarnings[common.Blobfuse2Version] {
+					msg = strings.TrimSpace(msg)
+					if len(msg) > 0 {
+						if !hasWarning {
+							fmt.Fprintf(stderr, "Following vulnerabilities were detected in your current version (%s):\n", common.Blobfuse2Version)
```

Comment:
The following

Reply:
Also should we log these?

Reply:
I am not sure if logging makes sense here, its more of user needs to notice this and take action. Dumping to logs will go unnoticed. Also version check is happening before the logging lib is initialized.

Reply:
Added the warnings to log